.env.schema

```
@@ -0,0 +1,30 @@
+
+# True to use the Feedback control
+# @type=boolean
+USE_FEEDBACK=true
+
+# Gets calulcated from the APP_MODE
+# @type=boolean
+IS_RUNTIME=not(eq($APP_MODE, build))
+
+# Gets calulcated from the APP_MODE
+# @type=boolean
+IS_BUILDTIME=not(eq($APP_MODE, prod))
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)